charts/spire/charts/spire-agent/values.yaml

```diff
@@ -86,7 +88,10 @@ waitForIt:
     repository: chainguard/wait-for-it
     # -- The image pull policy
     pullPolicy: IfNotPresent
-    version: latest-20230113
+    # -- This value is deprecated in favor of tag. (Will be removed in a future release)
+    version: ""
+    # -- Overrides the image tag
+    tag: latest
```

Comment:
I am not in favor using "latest" tag since they can change the content without our control, but in this specific scenario I assume the `chainguard` would not do anything that dramatic to break the test, so it's acceptable, I guess...